service/gcs/errors/errors.go

```diff
@@ -1,7 +1,24 @@
-package gcs
+package errors
 
 import (
 	"fmt"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+// Hresult is a type corresponding to the HRESULT error type used on Windows.
+type Hresult int32
+
+const (
+	HrUnexpected   = Hresult(-2147418113) // 0x8000FFFF
```

Comment:
If it's unsigned, then the JSON marshaler will marshal it as a positive integer, while the HCS expects it to be negative. I could maybe do some bitwise operations in the bridge to convert between a uint32 Hresult and the int32 the HCS is expecting, that just seemed like more work than was worth.